### PR TITLE
Remove deprecated quay image

### PR DESCRIPTION
**Pull request by msheiny**
This is linked to an internal effort to update docker containers that are afflicted by CVE-2019-3462, affecting `apt`. Defaulted to another image we recently updated.

Also removed CI checks here and reference to them in the docs. That is all deprecated logic on a deprecated role we are trying to phase out. It also doesn't help that we are on an older version of circle check config and were getting the following warning `Warning: Your project is using CircleCI 1.0 and will stop building on March 15, 2019. Update remaining projects to CircleCI 2.0 to continue building. Learn more.`

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -67,7 +67,7 @@ django_stack_npm_no_log: "{{ django_stack_global_no_log }}"
 django_stack_npm_global_pkgs: []
 django_stack_shell_commands: []
 # This is the tag for the node image used locally to prepare the code
-django_stack_node_ver: 6.11.0-alpine
+django_stack_node_ver: 10.15.1-alpine
```
Comment (conorsch) [inline defaults/main.yml]: The `node_ver` we'll have to test more... let's make sure to override it locally until we're ready to switch.

**Comment by conorsch** [comment]
Exciting CI failure here, @msheiny https://circleci.com/gh/freedomofpress/ansible-role-django/54 . Defer to you on shortest path, whether that's to update the ci-webserver container config and reuse that patched version, or continue to aim to transition the default

**Comment by msheiny** [comment]
> Exciting CI failure here, @msheiny https://circleci.com/gh/freedomofpress/ansible-role-django/54 . Defer to you on shortest path, whether that's to update the ci-webserver container config and reuse that patched version, or continue to aim to transition the default

Thanks @conorsch ! Just flagging from out of band chat. I removed CI checks here. I updated the repo settings and made some comments in a git commit and in the PR content above. Let me know any issues you foresee here.

**Comment by conorsch** [comment]
CI is listed as "pending" here, never to return. Checked the repo settings, and we're not actually requiring these status checks to pass, so I'm inclined to override and merge.

**Review by conorsch — APPROVED** [review]

**Comment by conorsch** [comment]
Did not have to override, merged cleanly after submitting approval.

**Pull request merged**